data/mappings/converters.hjson

```diff
@@ -0,0 +1,24 @@
+{
+    "promicro": {
+        "bit_c_pro": {},
+        "blok": {},
+        "bonsai_c4": {},
+        "elite_pi": {},
+        "kb2040": {},
+        "promicro_rp2040": {
+            "aliases": [
+                "Frood"
```

Comment:
```suggestion
                "frood"
```
The capital letter would be rejected by the suggested JSON schema.

Reply:
This block was initially for testing, and likely to be implemented as part of a "Bastardkb standard" mentioned in 19470. So for now, ive removed the block.